proparse/src/main/java/org/prorefactor/core/schema/ITable.java

```diff
@@ -3,7 +3,7 @@
 import java.util.List;
 import java.util.SortedSet;
 
-import org.prorefactor.treeparser.SymbolScopeRoot;
+import org.prorefactor.treeparser.TreeParserRootSymbolScope;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.prorefactor.treeparser.TreeParserRootSymbolScope'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AUselessImportCheck)